python/pipeline/utils/performance.py

```diff
@@ -407,14 +407,17 @@ def parallel_motion_stack(chunks, results, raster_phase, fill_fraction, skip_row
         # Apply anscombe transform
         field = 2 * np.sqrt(field - field.min() + 3 / 8)
 
-        # Compute initial template by averaging 10 frames that correlate highly with middle one
-        num_frames = field.shape[-1]
-        frames = np.reshape(field[skip_rows:-skip_rows, skip_cols:-skip_cols], (-1, num_frames)) # num_pixels x num_frames
+        # Compute correlation matrix between frames (ignoring edges)
```

Comment:
These reshapings are not necessary when adopting np.tensordot instead of np.dot.